src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -534,6 +534,7 @@ def _configure_db_dw_create_params(
         # --read-replica-count was accidentally included in previous releases and
         # therefore is hidden using `deprecate_info` instead of `ignore`
         arg_ctx.ignore('read_scale')
+        arg_ctx.ignore('high_availability_replica_count')
         arg_ctx.argument('read_replica_count',
                          options_list=['--read-replica-count'],
                          deprecate_info=arg_ctx.deprecate(hide=True))
```

Comment:
Do we need to remove this?

Reply:
I didn't want to remove this in case customers still try to use old param.  I don't see any problem in leaving it, but if you think it needs to be removed I will.  Thank you.